### PR TITLE
test: allowed systemd user journal corruption messages

### PR DESCRIPTION
**Pull request by jelly**
This happens on packit's Fedora 37.

### CODE REVIEW

File: test/browser/run-test.sh [hunk 1]
```diff
@@ -56,6 +56,11 @@ if [ "${TEST_OS#centos-8}" != "$TEST_OS" ]; then
     TEST_ALLOW_JOURNAL_MESSAGES="${TEST_ALLOW_JOURNAL_MESSAGES},couldn't create runtime dir: /run/user/1001: Permission denied"
 fi
 
+# HACK: https://github.com/systemd/systemd/issues/24150
+if [ "$ID" = "fedora" ]; then
+       TEST_ALLOW_JOURNAL_MESSAGES="Journal file /var/log/journal/*/user-1000@*.journal corrupted, ignoring file .*"
```
Comment (martinpitt) [inline test/browser/run-test.sh:61]: You need to append, like the other cases do. Please add the missing `${TEST_ALLOW_JOURNAL_MESSAGES},`

**Review by martinpitt — CHANGES_REQUESTED** [review]
Ah, that's an even nicer way to do that than adding a nasty to bots/machine_core!

File: test/browser/run-test.sh (hunk 1 above)
Comment (jelly) [reply test/browser/run-test.sh:61]: Oh I am so silly, yes!

**Review by martinpitt — APPROVED** [review]
Cheers! Please trigger the required test.

**Comment by jelly** [comment]
```
# Result testBasic (__main__.TestTeam.testBasic) failed
```
So this now also fails in F37 or flakes...

**Comment by jelly** [comment]
```
Error: Connection activation failed: IP configuration could not be reserved (no available address, timeout, etc.)
```

Is the issue.

**Comment by martinpitt** [comment]
@jelly The Team test has been failing for ages (and a retry works), that's not new.

**Comment by jelly** [comment]
> @jelly The Team test has been failing for ages (and a retry works), that's not new.

Yup, but I still want to get the flake away ;)

**Pull request merged**